ibtk/include/ibtk/AppInitializer.h

```diff
@@ -11,11 +11,15 @@
 //
 // ---------------------------------------------------------------------
 
+/////////////////////// INCLUDE GUARD ////////////////////////////////////
```

Comment:
I hate making this comment, but ...

I think that all of the comments in this style have the same width, and have the text start from the same column. Is it easy to stick with that?

Reply:
This would be pretty easy to fix with sed: I think we have two different lengths in the headers. Do you want the whole thing to be 80 characters or something else?

Reply:
My point is I am handy enough with sed that we can do whatever we want here - just tell me how long you want the ascii art guards to be and where the text will start and I can fix all of them in a few minutes.

Reply:
If there is a simple script that can easily standardize, I think that I don't care so much what the width is.

I _think_ that the most common is 78 characters, with the first block of slashes being 31 characters wide. (Why? I don't know!) Maybe stick with that to avoid additional changes to comments?

Reply:
Sure. Fixed!